bin/yarn

```diff
@@ -2,5 +2,9 @@
 
 set -e
 
-IMAGE_NAME=${IMAGE_NAME:-codeclimate/codeclimate-eslint}
-docker run --rm --volume "$PWD:/usr/src/app" "$IMAGE_NAME" sh -c "cd /usr/src/app && yarn $*"
+docker run \
+  --rm \
+  --user "$(id -u):$(id -g)" \
```

Comment:
Just pushed up a commit with a conditional.

@maxjacobson Would you mind trying it out?